src/components/Stakeholders/StakeholderDetail/Groups.js

```diff
@@ -64,7 +70,11 @@ function StakeholderGroups({ data, render, tall }) {
             <div className={classes.flex}>
               {stakeholder[type]?.icon}
               <Typography variant="h6">
-                {stakeholder[type]?.label} ({data[type]?.length || 0})
+                {stakeholder[type]?.label} (
+                {data[type]?.length ||
+                  (type === 'parents' && parent.length) ||
```

Comment:
Given we're doing multiple equality comparisons of `type`, it would be best to create some valid `const` or `enum` values rather than using string literals.

Reply:
I suggest we put aside this PR, until we decide on my post https://github.com/Greenstand/treetracker-admin-client/issues/754#issuecomment-1223653644
If the child element has a field of "parent" type "array" with "relation_id" inside, then the solution may be simpler